packages/excalidraw/element/textElement.ts

```diff
@@ -62,21 +62,26 @@ export const redrawTextBoundingBox = (
 
   boundTextUpdates.text = textElement.text;
 
-  if (container) {
-    maxWidth = getBoundTextMaxWidth(container, textElement);
+  if (container || !textElement.autoResize) {
+    maxWidth = container
+      ? getBoundTextMaxWidth(container, textElement)
+      : textElement.width;
     boundTextUpdates.text = wrapText(
       textElement.originalText,
       getFontString(textElement),
       maxWidth,
     );
   }
+
   const metrics = measureText(
     boundTextUpdates.text,
     getFontString(textElement),
     textElement.lineHeight,
   );
 
-  boundTextUpdates.width = metrics.width;
+  if (textElement.autoResize) {
```

Comment:
```suggestion
// Note: for bound text containers, the autoresize will always be set to true
  if (textElement.autoResize) {
```

For bound text containers the attribute will also be true right? In that case lets add a comment